docs/DeveloperGuide.md

```diff
@@ -365,20 +365,24 @@ Refer to the guide [here](./DevOps.md)
 
 **Target user profile**:
 
+* university students
 * take multiple modules
-* want to easily track their weekly calendar
-* want to be notified about important academic dates coming up
-* want to have their personal todos & events integrated with their academic calendar
+* want to easily know the events lined up in their week ahead
+* prefer to have everything in one application
+* want to have their personal tasks integrated with their academic calendar
```

Comment:
I suggest we can change this to something like: "want to have their personal todos integrated with the academic calendar"